protocols/bftcosi/bftcosi.go

```diff
@@ -61,18 +62,11 @@ type ProtocolBFTCoSi struct {
 	// channel for challenge during the commit phase
 	challengeCommitChan chan challengeCommitChan
 	// channel for response
-	responseChan chan responseChan
+	responseChan chan []responseChan
```

Comment:
That's the SDA-way of saying that it has to wait for all children before sending the messages.